### PR TITLE
fix: Fix nil values and int warning for parent team

### PR DESCRIPTION
**Pull request by Abhi347**
Fix nil values error in create team method

### CODE REVIEW

**Review by annelled — APPROVED** [review]

File: .sdkmanrc [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Enable auto-env through the sdkman_auto_env config
```
Comment (annelled) [inline .sdkmanrc:1]: Why is this needed?

Comment (Abhi347) [reply .sdkmanrc:1]: This is to force sdkman to switch to Java 11 to run this project. Until we make this project compatible with Java 17 (or higher), this is needed. It's easy to forget that this project runs on Java 11. May be this should be includes in the README too.
